src/containers/ExpandedPostModal.js

```diff
@@ -10,19 +10,28 @@ import Post from '../components/Post'
 export default class ExpandedPostModal extends React.Component {
   static propTypes = {
     dispatch: func,
+    socket: object,
     commentId: string,
     post: object
   }
 
   componentDidMount () {
-    const { commentId, post: { id }, dispatch } = this.props
+    const { commentId, post: { id, numComments }, dispatch, socket } = this.props
     dispatch(fetchComments(id, {offset: 3}))
     .then(() => {
       if (commentId) {
         const wrapper = document.getElementById(modalWrapperCSSId)
         scrollToComment(commentId, wrapper)
       }
     })
+    socket.on(`p/${id}/comment_added`, function (){
+      dispatch(fetchComments(id, {offset: numComments, refresh: true}))
```

Comment:
I'd much rather just pull down the new comment and then append it to the
comments array. Shouldn't be any problem with doing that.

On Mon, Aug 22, 2016 at 12:54 PM Connor Turland <notifications@github.com>
wrote:

> In src/containers/ExpandedPostModal.js
> <https://github.com/Hylozoic/hylo-redux/pull/107#discussion_r75746229>:
>
> >      dispatch(fetchComments(id, {offset: 3}))
> >      .then(() => {
> >        if (commentId) {
> >          const wrapper = document.getElementById(modalWrapperCSSId)
> >          scrollToComment(commentId, wrapper)
> >        }
> >      })
> > +    socket.on(`p/${id}/comment_added`, function (){
> > +      dispatch(fetchComments(id, {offset: numComments, refresh: true}))
>
> @levity <https://github.com/levity> could you comment on this approach of
> just using a message to notify that there's a new comment, and then getting
> the client side to fetch it? We could pass it down, but it feels like that
> would be weird, because then we'd have to append it into the comments array
> or something?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Hylozoic/hylo-redux/pull/107/files/9e098decd23c544f34bfa093e9bfab92269cca19#r75746229>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAL0g7NYt9blTuIba2GynUmNpoqzLC5Zks5qif5ngaJpZM4JqN01>
> .
>